RealmSwift-swift2.0/ObjectSchema.swift

```diff
@@ -74,6 +74,6 @@ public final class ObjectSchema: CustomStringConvertible {
 extension ObjectSchema: Equatable {}
 
 /// Returns whether the two object schemas are equal.
-public func ==(lhs: ObjectSchema, rhs: ObjectSchema) -> Bool {
+public func == (lhs: ObjectSchema, rhs: ObjectSchema) -> Bool {
```

Comment:
The style elsewhere appears to be to have no space between function name and opening parenthesis. Is the style intentionally different for operators, or is this being confused by a rule to have spaces around operators in other contexts?